Source/Csla/DataPortalClient/LocalProxy.cs

```diff
@@ -30,19 +30,18 @@ public LocalProxy(ApplicationContext applicationContext, LocalProxyOptions optio
       ApplicationContext = applicationContext;
       Options = options;
 
-      if (Options.CreateScopePerCall)
+      if (Options.CreateScopePerCall && ApplicationContext.ExecutionLocation == ApplicationContext.ExecutionLocations.Server)
```

Comment:
I am thinking this is not right.

I think a new scope should be created only if the _logical_ execution location is Client. After that first call, the data portal will have set the logical execution location to server, and subsequent calls through LocalProxy would then not create a new scope.

Reply:
Gosh I thought that but didn't say anything!  Ugh - trust your gut Sean.  Glad you caught it.

Reply:
That is such a nice feature being able to distinguish the diff between logical and physical :-)